check_sites.py

```diff
@@ -12,12 +12,44 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 from RwsCheck import RwsCheck
+import difflib
 import json
 import getopt
 import sys
 import os
 from publicsuffix2 import PublicSuffixList
 
+def load_rws_file_as_json(input_file, with_format):
+    """Attempts to load `input_file`, parse it as JSON, and validate formatting if `with_format` is true.
+
+        Returns a tuple of the JSON dict and None if there were no errors,
+        or None and the error message if there was an error.
+
+        Args:
+            input_file: string
+            with_format: bool
+        Returns:
+            Tuple[Dict|None, string|None]
+    """
+    with open(input_file) as f:
+        loaded_file = f.read()
```

Comment:
IMO pathlib is nicer here.
```suggestion
    loaded_file = Path(input_file).read_text()
```